builtins/xrootd/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ endif()
 ExternalProject_Add(
     XROOTD
     URL ${XROOTD_SRC_URI}
-    URL_HASH SHA256=56a29c88232f2f384e151b148fcaaa8d8db5c5fdc4615193978c8f4f3a99663c
+    URL_HASH SHA256=ba47813b216ccb61a2a54cf683510a74441861367febd730ab5d814ffa0cb376
```

Comment:
Need to wait to get the SHA from https://lcgpackages.web.cern.ch/tarFiles/sources/ once `xrootd-5.5.0.tar.gz` is available there. @simonmichal and @abh3 can you ping here when it is up?

Reply:
@pcanal @Axel-Naumann given your comments and help on PR #10725 do you know who uploaded `xrootd-5.4.3.tar.gz` and who we should ask to add `xrootd-5.5.0.tar.gz`?

Reply:
@andresailer can you either upload this or let me know where you take it from so I do it myself? Bertrand (who I assume would know) is on leave.

Reply:
I'm assuming that it is just taken (hopefully) from the GitHub release page: https://github.com/xrootd/xrootd/releases/tag/v5.5.0

```console
$ curl -sLO https://github.com/xrootd/xrootd/archive/refs/tags/v5.5.0.tar.gz
```

Reply:
There is also a source tarball at https://xrootd.slac.stanford.edu/2022/08/26/announcement_5_5_0.html and they are not (always) the same.

Reply:
The file is there now
```
curl -s https://lcgpackages.web.cern.ch/tarFiles/sources/xrootd-5.5.0.tar.gz | sha256sum 
e01d83997e428580aadefd4b18cc6246e97a1941dfceff3726ffd0c40921de34  -
```

Reply:
Thanks @andresailer! I've updated and rebased this now.

> and they are not (always) the same.

Hm, interesting. Do you know why? If this is something that can be discussed with the XRootD team in the future it would be good to avoid having variations between these.

Reply:
The github tarball is (was?) missing source code from git submodules.